app/models/test.rb

```diff
@@ -6,8 +6,10 @@ class Test < ActiveRecord::Base
 
   accepts_nested_attributes_for :answers
 
-  after_create :count_result
+  after_create :count_and_paste_result_into_test_table
 
-  resourcify#rolify
+  # include ZdubzdabHola::CountResult::LocalInstanceMethods
+  include MyModule
```

Comment:
better create some module and inside class. 
including creates additional logic and somethimes you forget to include something. 
better call this method directly
SomeNamespace::MyCalculation.calculte